demo/api/src/db/fixtures/fixtures.command.ts

```diff
@@ -185,6 +184,7 @@ export class FixturesCommand extends CommandRunner {
                 pages.push(pagesForLevel);
             }
             this.logger.log(`Generated ${pagesCount} lorem ipsum pages for ${domain}`);
+            this.entityManager.clear();
```

Comment:
Do we need a flush here?

Reply:
I don't think so, we create the pages with persist and flush. the function updateNodeVisibility is also flushing inside.


This code is in the "updateNodeVisibility"
<img width="609" height="103" alt="Screenshot 2025-11-13 at 09 42 06" src="https://github.com/user-attachments/assets/deaa81ed-3089-4d4f-b30b-2690d2259ae5" />


Or should I refactor it so it's only persisting and flushing at the end?